services/core-web/src/components/mine/Tailings/MineTailingsTable.js

```diff
@@ -7,10 +7,10 @@ import {
   getITRBExemptionStatusCodeOptionsHash,
 } from "@common/selectors/staticContentSelectors";
 import * as Strings from "@common/constants/strings";
+import { Button } from "antd";
```

Comment:
I dont see any changes involving this button is it needed?

Reply:
Thanks for the review. Oh this one was at line 13. So yes it's being required for the lines 95 and 104.